src/pages/topics/[slug].tsx

```diff
@@ -1,7 +1,12 @@
 import type { NextPage, GetStaticProps, GetStaticPaths } from 'next';
 import { ParsedUrlQuery } from 'querystring';
 
+import styled from '@emotion/styled';
+import theme from '@mdb/flora/theme';
+import { GridLayout } from '@mdb/flora';
+
 import Hero from '../../components/hero';
+import Search from '../../components/search/search';
```

Comment:
btw what do you think about at some point we create alias for common file directories so that we are not repeatedly writing out .`./../../etc `?

Reply:
Like so: https://webpack.js.org/configuration/resolve/

Reply:
Yeah i think we should absolutely do that, importing components from the pages is getting cumbersome lol.